content/300-guides/100-performance-and-optimization/150-connection-management/200-configure-pg-bouncer.mdx

```diff
@@ -41,7 +41,18 @@ Error: undefined: Database error
 Error querying the database: db error: ERROR: prepared statement "s0" already exists
 ```
 
-To work around this issue, you must connect directly to the database rather than going through PgBouncer. How to achieve this depends on your setup or provider:
+To work around this issue, you must connect directly to the database rather than going through PgBouncer. To achieve this, you can use the [`directUrl`](/reference/api-reference/prisma-schema-reference#datasource:~:text=the%20shadow%20database.-,directUrl,-No) field in your [`datasource`](/reference/api-reference/prisma-schema-reference#datasource) <span class="api"></span> block.
 
-- [Connecting directly to a PostgreSQL database hosted on Digital Ocean](https://github.com/prisma/prisma/issues/6157)
-- [Connecting directly to a PostgreSQL database hosted on ScaleGrid](https://github.com/prisma/prisma/issues/6701#issuecomment-824387959)
```

Comment:
I think these links might actually be useful, but in a slightly different context: It seems they effectively "document" how PgBouncer works on specific providers. Might be worth pulling into the documentation at the bottom as a "PgBouncer with different database providers" so it is easier for people to figure out that detail. (Heroku and Supabase would be other obvious candidates to list there.)